tests/unit/cluster/replica-migration.tcl

```diff
@@ -187,6 +189,7 @@ proc test_nonempty_replica {type} {
 
         # Wait for the cluster to be ok.
         wait_for_condition 1000 50 {
+            [R 4 cluster slots] eq [R 7 cluster slots] &&
```

Comment:
The nodes in CLUSTER SLOTS are sorted now, including replicas, right?

Reply:
yes, CLUSTER SLOTS is sorted by slot, replicas also are sorted now.